packages/select/docs/FilterMultiSelect.stories.scss

```diff
@@ -0,0 +1,20 @@
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/shadow";
+
+.row {
+  display: inline-grid;
+  grid-template-columns: 1fr auto;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  box-shadow: $shadow-large-box-shadow;
+  border-radius: 7px;
```

Comment:
Would be good to use the design token here in case people copy+paste, but we can follow that up.